sdk/storage/ci.yml

```diff
@@ -11,6 +11,7 @@ trigger:
       - eng/
       - CMakeLists.txt
       - sdk/core
+      - sdk/identity
```

Comment:
Does Storage need the identity package?

Reply:
Yes - they use ClientSecretCredential, that sits in Identity.
In .NET SDK, it sits in Identity, too.
And that is fine, when you think that Storage depends even on HTTP adapters, Identity dependence is much more "native", natural, simple and lightweight dependency.

Reply:
I think my answer resolves the conversation.